packages/jest-runner/package.json

```diff
@@ -8,7 +8,10 @@
   },
   "license": "MIT",
   "main": "build/index.js",
+  "types": "build/index.d.ts",
   "dependencies": {
+    "@jest/core": "^24.1.0",
```

Comment:
`jest-runner` cannot depend on `core`, that's a cyclic dependency. It's not even TS, what are you importing from it?